### PR TITLE
Show beta warning for transfers going through AT API

### PR DESCRIPTION
**Pull request by NunoAlexandre**
### Changes

- Drop `useDryRunValidation` - The PJS validation is not useful enough to justify a two signature flow
- [off-Improve naming of the from/to swap feature
- Make ActionBanner more flexible by making the button optional
- ⭐  Show banner with warning for transfers that will be run through AT API

### CODE REVIEW

File: app/public/wip.png
Comment (Victor-Salomon) [inline app/public/wip.png:1]: 🤩

File: app/src/components/Transfer.tsx [hunk 1]
```diff
@@ -1,14 +1,13 @@
...
 import useErc20Allowance from '@/hooks/useErc20Allowance'
 import useSnowbridgeContext from '@/hooks/useSnowbridgeContext'
 import useTransferForm from '@/hooks/useTransferForm'
 import { resolveDirection } from '@/services/transfer'
-import { getRecipientAddress } from '@/utils/address'
 import {
   getAllowedDestinationChains,
   getAllowedSourceChains,
   getAllowedTokens,
+  getRoute,
```
Comment (Victor-Salomon) [inline app/src/components/Transfer.tsx:10]: Let's be careful with file namings that use the word: `route`
For example: We have a `routes.ts `in utils. 

**`route`** is a word "owned" by Next.js as part of its architecture. 

There is no proper issue for now. 
This is more like a global warning to keep in mind :)

**Review by Victor-Salomon — APPROVED** [review]
Looks good to me. 
Warning text is simple and smart 👌

File: app/src/components/Transfer.tsx (hunk 1 above)
Comment (NunoAlexandre) [reply app/src/components/Transfer.tsx:10]: I agree!

**Pull request merged**